plugins/AIAgents/Providers/NovaAct.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+declare(strict_types=1);
+
+namespace Piwik\Plugins\AIAgents\Providers;
+
+use Piwik\Tracker\Request;
+
+class NovaAct extends AgentAbstract
+{
+    public function getId(): string
+    {
+        return 'NovaAct';
+    }
+
+    public function isDetectedForTrackerRequest(Request $trackerRequest): bool
+    {
+        $userAgent = $trackerRequest->getUserAgent();
+
+        return stripos($userAgent, 'NovaAct') !== false;
```

Comment:
would being more precise in the string matching possible avoid some false positives?
```suggestion
        return stripos($userAgent, 'Agent-NovaAct') !== false;
```